# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1310829400 25200
#      Sat Jul 16 08:16:40 2011 -0700
# Node ID 482cc053281bb317809310a5d636f4a65be7005e
# Parent  9f4b48b93f6d97704adbc1f1bf954b7c64670ec6
codesprint: Add 2 demos for polar.m

* polar.m : Add 2 rose demos.

diff --git a/scripts/plot/polar.m b/scripts/plot/polar.m
--- a/scripts/plot/polar.m
+++ b/scripts/plot/polar.m
@@ -206,8 +206,20 @@ function retval = __plr2__ (h, theta, rh
     else
       error ("__plr2__: invalid data for plotting");
     endif
   else
     error ("__plr2__: invalid data for plotting");
   endif
 
 endfunction
+
+
+%!demo
+%! theta = linspace (0, 2*pi, 1000);
+%! rho = sin (7*theta);
+%! polar (theta, rho);
+
+%!demo
+%! theta = linspace (0, 10*pi, 1000);
+%! rho = sin (5/4*theta);
+%! polar (theta, rho);
+
